app/assets/javascripts/users.js

```diff
@@ -29,3 +29,26 @@ function get_select_values(select) {
 
   return result;
 }
+
+function test_mail(item, id) {
+  $(item).addClass("disable-link");
+  $('#test_indicator').show();
+  var data = '{"user_email" : "' +  $("#user_mail").val() + '"}';
+  var param = JSON.parse(data);
+  $.ajax({
+    url: foreman_url("users/" + id + "/test_mail"),
```

Comment:
When on /users/24/edit, the request being generated is to `PUT "/users/24/users/24/test_mail"`, what's happening?